opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -227,7 +227,19 @@ namespace Opm {
 
         using namespace std::placeholders;
         const auto handler = std::bind( handleKW, _1, schedule, props, n_xyz );
+
+#ifdef _MSC_VER
+        for (auto& x : section)
+        {
+            auto keywords = handler(x);
+            for (auto& keyword : keywords)
+            {
+                this->keywords.push_back(keyword);
+            }
+        }
+#else
         this->keywords = fun::concat( fun::map( handler, section ) );
```

Comment:
> Leftpad has nothing to do with this

really? I agree that the NPM packaging issues have nothing to do with this, but the fact that this community felt like it was a good idea to introduce abstraction for two lines of code **has** quite a bit of relevance in the current context IMO. 

> abstractions reduce the mental load

IMO they first increase the mental load because you have to get familiar with them. They are only beneficial when they are so common that you don't need to think about them anymore...



>>   most places where these are used are in no way performance critical, I suppose?
>
> It's not just for efficiency, it's for generality and applicability. Want to populate a vector? Fine. Want to populate a list? Fine, same code. Want to populate a std::map? No problem. Want to not populate anything, but rather evalute-and-reduce? Got your back.

yeah, and in all cases I need to look up in the implementation what it means _exactly_ to e.g. concatenate a map and a vector.

>> if we were talking lisp or scheme, I'd agree. But in C++, these are not standard constructs, and people are not used to the functional programming paradigm...
>
> This is just not true.

let's compromise by saying that in C++ it is in the eye of the beholder and that at least the "old c++ dogs" are not very familiar with it ;)